packages/yoga/src/Theme/themes/BaseTheme.js

```diff
@@ -320,6 +320,38 @@ const BaseTheme = ({ primary, secondary, tertiary }) => {
           },
         },
       },
+      event: {
+        icon: {
+          fill: colors.dark,
+        },
+        info: {
+          padding: spacing.medium,
+          name: {
+            fontWeight: fontWeights.semibold,
+            padding: {
+              bottom: spacing.xsmall,
+            },
+          },
+          place: {
+            color: colors.gray[8],
+            padding: {
+              bottom: spacing.xsmall,
+            },
+          },
+        },
+        date: {
+          radius: radii.rounded,
+          weekday: {
```

Comment:
`weekday` isn't all days of the week. Maybe `daysOfWeek` would be better. What do you think?

Reply:
![image](https://user-images.githubusercontent.com/11219999/72619682-9af77e80-391c-11ea-9a21-68fc92c52302.png)

 Agree with u